src/views/Property/PropertyView.jsx

```diff
@@ -201,9 +189,20 @@ const Property = () => {
       });
   }
 
-  const removeTenant = (id) => {
+  const removeTenant = (tenant) => {
+    setTenantToRemove(tenant);
+    setConfirmTenantRemoval(true);
+  }
 
-    getProperty()
+  const handleTenantConfirmButton = () => {
+    const leaseId = tenantToRemove.lease.id;
+    axios
+      .delete(`${process.env.REACT_APP_PROXY}/api/lease/${leaseId}`)
+      .then(res => getProperty())
+      .then(setConfirmTenantRemoval(false))
+      .then(setEditingStatus(false))
+      .then(Toast("Tenant successfully removed"))
```

Comment:
Once the response is received (line 201), further calls are not dependent on the completion of those prior. In other words, I think this section needs only one ".then".
However, I haven't run this code yet. If I am off-base here please let me know.

Reply:
This has been corrected.  Thank you!

Reply:
I'm not seeing this change